### PR TITLE
Busco list datasets

### PR DESCRIPTION
**Pull request by dorien-er**
## Description

add component to list available busco datasets (that can be used in the busco component)

## Issue ticket number

Closes #xxxx 

## Checklist before requesting a review

- [x] I have performed a self-review of my code

- [x] Conforms to the [Contributing guidelines](https://github.com/viash-hub/base/blob/main/CONTRIBUTING.md)

- [x] Proposed changes are described in the [CHANGELOG.md](https://github.com/viash-hub/base/blob/main/CHANGELOG.md)

- [x] I have tested my code with `viash ns test --parallel -q <name or namespace>`

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [x] New functionality
  - [ ] Major changes
  - [ ] Minor changes
  - [ ] Documentation
  - [ ] Bug fixes

### CODE REVIEW

**Comment by dorien-er** [comment]
this is a supporting component that's referenced in busco/busco, is it useful to add the help text here again, since they're already present in busco/busco?

**Comment by rcannood** [comment]
> this is a supporting component that's referred to in busco/busco, is it useful to add the help text here again, since they're already present in busco/busco?

One possibility is to add it to one entry but structure it a little bit, e.g.

* Add busco: <description>
  - busco run (#...)
  - busco list (#...)
  - busco download (#...)

**Comment by dorien-er** [comment]
> > this is a supporting component that's referred to in busco/busco, is it useful to add the help text here again, since they're already present in busco/busco?
> 
> One possibility is to add it to one entry but structure it a little bit, e.g.
> 
> * Add busco: 
>   
>   * busco run (#...)
>   * busco list (#...)
>   * busco download (#...)
> 
> WDYT?

Sounds good... But what about the help.txt file - doesn't seem useful to have that seem file in all 3 subcomponents, shall I just leave it under the main component only?

**Review by rcannood — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,7 +8,9 @@
 
 * `fastp`: An ultra-fast all-in-one FASTQ preprocessor (PR #3).
 
-* `busco`: Assess genome assembly and annotation completeness with single copy orthologs (PR #6).
+* `busco`: 
+    - `busco/busco`: Assess genome assembly and annotation completeness with single copy orthologs (PR #6).
+    - `busco_list_datasets`: Lists available busco datasets (PR #18)
```
Comment (rcannood) [inline CHANGELOG.md:13]: Would this make sense?

```suggestion
    - `busco/busco_run`: Assess genome assembly and annotation completeness with single copy orthologs (PR #6).
    - `busco/busco_list_datasets`: Lists available busco datasets (PR #18)

**Review by rcannood — CHANGES_REQUESTED** [review]

**Review by rcannood — APPROVED** [review]

**Pull request merged**